priv/repo/migrations/20220820164826_delete_orphaned_data_from_tables_missing_references.exs

```diff
@@ -7,29 +7,25 @@ defmodule Ret.Repo.Migrations.DeleteOrphanedDataFromTablesMissingReferences do
   """
```

Comment:
This looks like just a formatting change, yes?

Reply:
Bingo